source/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication.md

```diff
@@ -7,7 +7,9 @@ category:
 ---
 SSL is a standard for establishing an encrypted link between your Pantheon site and a client (e.g. web browser). You should enable SSL on a custom domain, e.g., www.example.com, if you are transmitting any sensitive data. Loading a valid OpenSSL certificate into a Pantheon environment provisions an SSL load balancer with a dedicated IP address, allowing secure communication over HTTPS.
 
-**Note**: Enable SSL before updating DNS. SSL for custom domains is available for Professional plans and above.
+**Important**: Enable SSL before updating DNS. SSL for custom domains is available for Professional plans and above.
+
+**Note**: All traffic within the Pantheon infrastructure, from Varnish to application containers, is encrypted.
```

Comment:
How about moving the new note to the first paragraph? "...allowing secure communication over HTTPS. All traffic within the Pantheon Infrastructure, from Varnish to application containers, is encrypted."

I feel like having "Important" and then "Note" gives more noise and detracts from the fact that they need to enable SSL before updating DNS.